specification/delegates/RegisterDelegate/RegisterDelegateTransactionDto.yaml

```diff
@@ -1,5 +1,5 @@
 allOf:
-  - $ref: "../../common/transactions/QueuedTransaction.yaml"
+  - $ref: "../../common/transactions/RegisterDelegateTransaction.yaml"
```

Comment:
@martiliones 
When registering a new delegate with [POST /api/delegates](https://github.com/Adamant-im/adamant/wiki/API-Specification#register-delegate-transaction), it responds with a [QueuedTransaction](https://github.com/Adamant-im/adamant/wiki/API-Specification#get-specific-queued-transaction) and not a final transaction (`RegisterDelegateTransaction`) that is already stored in the blockchain.


You should either modify the `QueuedTransaction.yaml` or extend the type in the current file